app/controllers/api/predictions_controller.rb

```diff
@@ -0,0 +1,42 @@
+module Api
+  class PredictionsController < ApplicationController
```

Comment:
It might be worth looking into [respond_to](http://apidock.com/rails/v3.2.13/ActionController/MimeResponds/respond_to) and [respond_with](http://apidock.com/rails/v3.2.13/ActionController/MimeResponds/respond_with) for this controller. I think it will take care of a bunch of stuff for you like response statuses and propagating errors.